crates/hc_bundle/tests/fixtures/my-app-deferred/dna/dna.yaml

```diff
@@ -0,0 +1,11 @@
+manifest_version: '1'
```

Comment:
Are these test fixtures a merge error? They don't seem to be relevant to the PR at hand, and are not used by any tests.